tests/cmake/FindNetCDF.cmake

```diff
@@ -0,0 +1,78 @@
+# - Find NetCDF
```

Comment:
Is this needed?

Reply:
If I remove it on my system it still works, but my recollection is that on larger more diverse systems (HPCs) it is required. I think leave in for now.